features/core-download.feature

```diff
@@ -18,6 +18,12 @@ Feature: Download WordPress
     And I run `cd inner && wp core download`
     Then the inner/wp-settings.php file should exist
 
+    When I try `WP_CLI_STRICT_ARGS_MODE=1 wp core download --path=inner`
```

Comment:
So if you try `WP_CLI_STRICT_ARGS_MODE=1 wp core download --path=test` what happens?

> Are you working on Windows?

How dare you!

I've pushed a non-WP_CLI_STRICT_ARGS_MODE case as well.